source/php-libraries.txt

```diff
@@ -60,7 +64,11 @@ Framework Integrations
     library and includes an abstract model, a helper class for handling bulk
     writes, database wrappers for Laravel and Lumen, and some necessary helper
     functions.
-    
+
+  - `Xenus Laravel <https://github.com/abellion/xenus-laravel>`_ integrates
+    Xenus to the Laravel and Lumen frameworks, adding failed jobs and migrations
```

Comment:
I'd rewrite as "integrates Xenus ODM with the Laravel...". Aside from s/to/with/, referring to it as "Xenus ODM" hints at the library's purpose without expecting the reader to see the earlier bullet on the page.